docs/import/create_disaster_lib.js

```diff
@@ -1,44 +1,73 @@
 export {createDisasterData};
-export {incomeKey, snapKey, sviKey, totalKey};
+export {BUILDING_COUNT_KEY, incomeKey, snapKey, sviKey, totalKey};
 // For testing
-export {assetDataTemplate};
+export {deepCopy, flexibleAssetData, stateAssetDataTemplate};
 
 const snapKey = 'HD01_VD02';
 const totalKey = 'HD01_VD01';
 const sviKey = 'RPL_THEMES';
 const incomeKey = 'HD01_VD01';
+const BUILDING_COUNT_KEY = 'BUILDING COUNT';
 
-// Has all the basic fields needed for the score asset to be created: SNAP, SVI,
-// and income together with the columns of each, and optional damage asset path.
-// The default values for column names here are taken from Census American
-// FactFinder and CDC tables.
+const commonAssetDataTemplate = Object.freeze({
+  damageAssetPath: null,
+  damageLevelsKey: null,
+  noDamageValue: null,
+});
+
+const stateBasedDataTemplate = Object.freeze({
+  scoreBoundsCoordinates: null,
+  blockGroupAssetPaths: {},
+  snapData: {
+    paths: {},
+    snapKey,
+    totalKey,
+  },
+  sviAssetPaths: {},
+  sviKey,
+  incomeAssetPaths: {},
+  incomeKey,
+  buildingAssetPaths: {},
+  states: null,
```

Comment:
what is the difference between
> myKey: null,

and

> myKey,

?

Reply:
JavaScript destructuring: the second one means `'myKey': myKey,`